### PR TITLE
Add logger

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Maybe static logger would be a better idea 🤔

File: src/logger.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include "logger.hpp"
+
+std::unique_ptr<Logger> g_logger = std::make_unique<Logger>();
```
Comment (akondas) [inline src/logger.cpp:3]: What kind of magic is this?

**Review by akondas — COMMENTED** [review]
why do we need a logger?

File: src/logger.cpp (hunk 1 above)
Comment (akondas) [reply src/logger.cpp:3]: singleton c++ style :+1:

**Review by akondas — APPROVED** [review]

**Pull request merged**